src/dct/enum.lua

```diff
@@ -219,4 +219,25 @@ enum.event = {
 		--]]
 }
 
+enum.kickCode = {
+	["NOKICK"]  = 0,
+	["UNKNOWN"] = 1,
+	["SETUP"]   = 2,
+	["EMPTY"]   = 3,
+	["DEAD"]    = 4,
+	["LOADOUT"] = 5,
+	["MISSION"] = 6,
+}
+
+enum.kickReason = {
+	[enum.kickCode.NOKICK] =
+		"no kick requested, should never see this report bug",
```

Comment:
```suggestion
		"No kick was requested. This was an error. Please report a bug.",
```

Reply:
This doesn't make sense the message sent will be:

```
*** you have been kicked from slot(<slotid>) ***
   reason: <reason>
```

Where `<slotid>` is the ID of the slot, and `<reason>` is one of these reason messages, so if you are capitalizing for grammar sake it doesn't make sense in the entire message context. Also these messages need to be short. People are not going to read a book.

So I would change to:
```no kick requested, please report the bug.```